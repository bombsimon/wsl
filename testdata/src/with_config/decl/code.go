package pkg

func Fn() {
	a := 1
	b := 2
	c := 3
}
