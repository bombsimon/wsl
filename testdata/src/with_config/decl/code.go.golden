package pkg

func Fn() {
    var a = 1
    var b = 2
    var c = 3
}
